### ISSUE TITLE
sy_modification, not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2621]
```abap
PERFORM sdfs CHANGING sy-subrc.
FORM sdfs CHANGING a.
ENDFORM.
```
